README.md

```diff
@@ -1,2 +1,12 @@
 # UofT-DSI-Cohort5-ML-T1
 Team project for UofT-DSI cohort 5 Machine Learning Team 1
+
+
+
+
+## What are the risks and uncertainties?
+- High market volatility and unpredictability
+- Missing or inconsistent historical data
+- Excludes external factors (e.g., news, regulations)
+- Market behavior may change over time
+- Limited accuracy in short-term forecasting
```

Comment:
One more point can be fine-grained market sentiments, as the dataset ranges daily.